index.html

```diff
@@ -2,9 +2,335 @@
+            <p>Het coderen vereist een systematische en analytische benadering, waarbij studenten worden
+                aangemoedigd om logisch te redeneren en problemen op te lossen. Door het ontleden van complexe
+                problemen in kleinere componenten en het ontwikkelen van algoritmen, verbeteren studenten hun
+                kritisch denkvermogen. Ze leren problemen oplossen stap voor stap en evalueren de efficiëntie
+                en
+                effectiviteit van hun code <em>(Resnick et al., 2009)</em>.</p>
+            <h3 id="zelfmanagement">Zelfmanagement</h3>
+            pHet programmeren vereist nauwkeurigheid, doorzettingsvermogen en zelfdiscipline. Studenten leren
+            plannen, organiseren en prioriteiten stellen om hun projecten succesvol af te ronden. Door te
+            programmeren oefenen studenten zelfmanagementvaardigheden, zoals het stellen van doelen, het
+            plannen
+            van taken en het behouden van overzicht. Het succesvol oplossen van programmeerproblemen activeert
+            chemische reacties in de hersenen <em>(Balanskat & Englehart, 2014)</em>, waaronder het vrijkomen
+            van
+            serotonine (C<sub>10</sub>H<sub>12</sub>N<sub>2O</sub>), waardoor studenten een gevoel van
+            voldoening en welzijn ervaren.
+
+            <figure>
+                <img src="../../../../Working%20files/Sjoerd%20Jansz2/Wetenschappelijk%20artikel%20(HTML%20%26%20CSS%20les%201)/assets/serotonine_chart.png" alt="serotonine-grafiek">
+                <figcaption><strong>Figuur 3:</strong> Scheikundige representatie serotonine</figcaption>
+            </figure>
+
+            <h3 id="academische-vaardigheden">Academische vaardigheden</h3>
+            <p>Naast de directe educatieve voordelen, heeft het leren programmeren ook een positief effect op
+                academische vaardigheden, zoals leesvaardigheid, schrijfvaardigheid en probleemoplossend
+                vermogen.
+                Het coderen vergroot de cognitieve flexibiliteit en de capaciteit om complexe informatie te
+                verwerken. Het biedt studenten ook de mogelijkheid om vertrouwd te raken met technologische
+                tools
+                en concepten die essentieel zijn in de moderne wereld. Ze leren bijvoorbeeld over databases,
+                netwerken, gebruikersinterfaces en web technologieën, zoals <abbr>HTML</abbr> (HyperText
+                Markup
+                Language) en
+                <abbr>CSS</abbr> (Cascading Style Sheets). Deze technologische kennis kan hen helpen bij het
+                uitvoeren van
+                onderzoek, het creëren van multimedia-inhoud en het presenteren van informatie op digitale
+                platforms, zoals wordt gedaan op het wereldwijd bekende <a
+                        href="https://edhub.novi.nl/">EdHub</a> gebouwd door NOVI
+                Hogeschool
+                (zie
```

Comment:
Op deze regel ontbreekt een opening `<p>` tag. De regel begint met "pHet" in plaats van "<p>Het"